website/docs/intro/install/rpm.sh

```diff
@@ -2,14 +2,22 @@
 
 set -e
 
-if [ -f /usr/bin/zypper ]; then
+if command_exists "zypper"; then
```

Comment:
command_exists is not defined within this file.  You are trying to refer to a function defined inside the install-opentofu.sh script that lives in the get.opentofu.org repository.  You can pull a similar function into this script if need be.